src/Sylius/Component/Core/OrderProcessing/InventoryHandler.php

```diff
@@ -91,6 +91,7 @@ public function processInventoryUnits(OrderItemInterface $item)
      */
     public function holdInventory(OrderInterface $order)
     {
+        // TODO: handle multiple stock locations
```

Comment:
We are focusing in this PR just on bringing models as [described](https://github.com/Sylius/Sylius/pull/3233#issuecomment-139947841) before, services will be alived in next iteration.